airflow/triggers/base.py

```diff
@@ -51,7 +52,7 @@ class StartTriggerArgs:
 
 class BaseTrigger(abc.ABC, LoggingMixin):
     """
-    Base class for all triggers.
+    Base class for triggers used to defer tasks.
```

Comment:
can BaseEventTrigger also be used to defer task?

Reply:
Good point. Answer is yes so I guess I can rephrase it. Good catch!